site/content/specs.md

```diff
@@ -76,53 +76,53 @@ your use case (ex which language you are using to compile, a.k.a. target Wasm).
 <details><summary>Click to see the full list of supported WASI functions</summary>
 <p>
 
-| Function                | Status | Known Usage    |
-|:-----------------------:|:------:|---------------:|
-| args_get                |   ✅   | TinyGo         |
-| args_sizes_get          |   ✅   | TinyGo         |
-| environ_get             |   ✅   | TinyGo         |
-| environ_sizes_get       |   ✅   | TinyGo         |
-| clock_res_get           |   ❌   |                |
-| clock_time_get          |   ✅   | TinyGo         |
-| fd_advise               |   ❌   |                |
-| fd_allocate             |   ❌   |                |
-| fd_close                |   ✅   | TinyGo         |
-| fd_datasync             |   ❌   |                |
-| fd_fdstat_get           |   ✅   | TinyGo         |
-| fd_fdstat_set_flags     |   ❌   |                |
-| fd_fdstat_set_rights    |   ❌   |                |
-| fd_filestat_get         |   ❌   |                |
-| fd_filestat_set_size    |   ❌   |                |
-| fd_filestat_set_times   |   ❌   |                |
-| fd_pread                |   ❌   |                |
-| fd_prestat_get          |   ✅   | TinyGo,`fs.FS` |
-| fd_prestat_dir_name     |   ✅   | TinyGo         |
-| fd_pwrite               |   ❌   |                |
-| fd_read                 |   ✅   | TinyGo,`fs.FS` |
```

Comment:
removed the `fs.FS` thing I added at one point as we don't actually have an `fs.FS` abstraction. It is better to focus on language compilers.